compiler/const-manipulations.h

```diff
@@ -10,7 +10,9 @@
 #include "common/php-functions.h"
 #include "compiler/data/data_ptr.h"
 #include "compiler/data/define-data.h"
-#include "compiler/gentree.h"
+#include "compiler/data/var-data.h"
+#include "compiler/vertex-util.h"
+#include "compiler/compiler-core.h"
```

Comment:
We usually try to keep includes sorted (here and everywhere below, when gentree.h replaced by vertex-util.h)

Reply:
Fixed.